handlers/search.go

```diff
@@ -237,22 +244,18 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 		if utils.IsValidEnsDomain(search) {
 			ensData, _ = GetEnsDomain(search)
 			if len(ensData.Address) > 0 {
-				search = strings.Replace(ensData.Address, "0x", "", -1)
+				lowerStrippedSearch = strings.Replace(ensData.Address, "0x", "", -1)
```

Comment:
The result of this is not lower case. You need to call `strings.ToLower` again:
```suggestion
				lowerStrippedSearch = strings.ToLower(strings.Replace(ensData.Address, "0x", "", -1))
```